### ISSUE TITLE
Error building SCS

### ISSUE THREAD

**Issue by eddiemattia** [JuliaOpt/SCS.jl#97]
Hey, wondering if anyone can help me figure out why I am getting a build error for the SCS solver. The following message has come up each time I try to build it again after deleting the failing package. Any advice @staticfloat ? 

LoadError: Provider BinDeps.Binaries failed to satisfy dependency scs
while loading C:\Users\ematt\AppData\Local\JuliaPro-0.6.2.1\pkgs-0.6.2.1\v0.6\SCS\deps\build.jl, in expression starting on line 76

================================================================================

================================[ BUILD ERRORS ]================================

WARNING: SCS had build errors.

 - packages with build errors remain installed in C:\Users\ematt\AppData\Local\JuliaPro-0.6.2.1\pkgs-0.6.2.1\v0.6
 - build the package(s) and all dependencies with `Pkg.build("SCS")`
 - build a single package by running its `deps/build.jl` script

**Comment by eddiemattia** [comment]
I also get this message when I try using the package if it helps... 

ERROR: LoadError: SCS not properly installed. Please run Pkg.build("SCS") and restart julia
Stacktrace:
 [1] error(::String) at .\error.jl:21
 [2] include_from_node1(::String) at .\loading.jl:576
 [3] include(::String) at .\sysimg.jl:14
 [4] anonymous at .\<missing>:2
while loading C:\Users\ematt\AppData\Local\JuliaPro-0.6.2.1\pkgs-0.6.2.1\v0.6\SCS\src\SCS.jl, in expression starting on line

**Comment by mlubin** [comment]
We've replaced the build script, so give this a try again once the 0.4.0 release is merged. Please open a new issue if needed.

**Comment by jzazo** [comment]
What's the status? I see 0.4.0 has been tagged, but I get a similar compilation error with the following detail:
```
mkdir -p out
gcc -DCOPYAMATRIX -DDLONG -DLAPACK_LIB_FOUND -DCTRLC=1 -DMKL_ILP64 -DBLAS64 -fopenmp -g -Wall -Wwrite-strings -pedantic -O3 -funroll-loops -Wstrict-prototypes -I. -Iinclude -fPIC  -DCTRLC=1  -DCOPYAMATRIX=1  -shared -Wl,-soname,libscsdir.so -o out/libscsdir.so src/scs.o src/util.o src/cones.o src/cs.o src/linAlg.o src/ctrlc.o src/scs_version.o linsys/direct/private.o linsys/direct/external/ldl.o linsys/direct/external/amd_defaults.o linsys/direct/external/amd_1.o linsys/direct/external/amd_valid.o linsys/direct/external/amd_preprocess.o linsys/direct/external/amd_control.o linsys/direct/external/amd_post_tree.o linsys/direct/external/amd_order.o linsys/direct/external/amd_aat.o linsys/direct/external/amd_info.o linsys/direct/external/amd_global.o linsys/direct/external/amd_2.o linsys/direct/external/amd_postorder.o linsys/direct/external/amd_dump.o linsys/common.o  -lmkl_intel_ilp64 -lmkl_gnu_thread -lmkl_rt -lmkl_core -lgomp -lm -lrt
/usr/bin/ld: cannot find -lmkl_intel_ilp64
/usr/bin/ld: cannot find -lmkl_gnu_thread
/usr/bin/ld: cannot find -lmkl_rt
/usr/bin/ld: cannot find -lmkl_core
collect2: error: ld returned 1 exit status
make: *** [out/libscsdir.so] Error 1
```

I am trying to add the package on a cluster which requires modules to be loaded, but I don't know which ones to load for this case. To add Clp I had to load fortran modules, but I am clueless here. Thank you.

**Comment by jzazo** [comment]
I finally managed to get the packet added and built. I needed to load Openblas in the cluster, but there where several versions and only one worked. Additionally, I had to
```
$ cd SCS/deps/
$ rm -r src usr downloads
$ julia
julia> Pkg.build("SCS")
```
as mentioned [here](https://github.com/JuliaOpt/SCS.jl/issues/58#issuecomment-241874813). Thanks.
